src/napari_spatialdata/_interactive.py

```diff
@@ -77,7 +80,36 @@ def _onClick(self, text: str) -> None:
         elif self.elements_widget._elements[text] == "shapes":
             self._add_shapes(text)
 
+    def _update_visible_in_cs(self, event: Event) -> None:
```

Comment:
I would prefer if `cs` was written out as `coordinate_system` in the function name. I think this is much more clear.

Reply:
agree, long function name though hehe

Reply:
Resolved with [2ec5203](https://github.com/scverse/napari-spatialdata/pull/99/commits/2ec5203d9858db9952e7375f5374d042a4e06738)